library/core/src/str/converts.rs

```diff
@@ -14,6 +15,7 @@ use crate::{mem, ptr};
 ///
 /// [`&str`]: str
 /// [byteslice]: slice
+/// [`<str>::from_utf8`]: ../../std/primitive.str.html#method.from_utf8
```

Comment:
Is there a reason we can't use a rustdoc-generated link here? Does that not work for primitive types or is something else wrong with that?